packages/base-controller/src/ControllerMessenger.test.ts

```diff
@@ -361,6 +361,41 @@ describe('ControllerMessenger', () => {
     expect(selector.callCount).toStrictEqual(4);
   });
 
+  it('should throw subscriber errors in a timeout', () => {
+    const setTimeoutStub = sinon.stub(globalThis, 'setTimeout');
+    type MessageEvent = { type: 'message'; payload: [string] };
```

Comment:
I've never seen a type set in a runtime context before. Does this set it locally or does it set it globally? If globally, then I'm wondering if it would be more obvious to extract this to the top of the file.

Reply:
Hmm, I'm not sure. I was copying this from another test.

Reply:
Seemingly they are locally scoped, so this should work fine. I've tried playing around with this a bit in the TypeScript playground.